model_analyzer/triton/model/model_config.py

```diff
@@ -347,6 +347,11 @@ def set_composing_model_variant_name(self, composing_model_name: str,
 
         self._model_config = self.from_dict(model_config_dict)._model_config
 
+    def set_model_name(self, model_name: str) -> None:
```

Comment:
is there no easier way than a full from/to dict?

Reply:
There is, but it would require string parsing with search and replacement which seems unsafe/prone to error.